nx/include/switch/services/applet.h

```diff
@@ -468,3 +468,18 @@ u32 appletGetPerformanceMode(void);
 AppletFocusState appletGetFocusState(void);
 
 Result appletSetFocusHandlingMode(AppletFocusHandlingMode mode);
+
+/**
+ * @brief Stops forwaring the input to the foreground app. Works only in the Overlay applet context
```

Comment:
*forwarding

Reply:
periods

Reply:
Description seems completely different from cmd name?

Reply:
Yes, i couldn't understand the meaning at first but that's the only effect it has and works consistently to lock/unlock the input, makes sense if you think about it like "start displaying the quick action menu"